src/clean_macro_data.py

```diff
@@ -0,0 +1,54 @@
+import logging
+import os
+import re
+import argparse
+
+import yaml
+import pandas as pd
+import boto3
+
+from helpers import gen_helpers as gen_h
+
+logging.basicConfig(format="%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+logger = logging.getLogger(__name__)
+
+def run_ipo(args):
+    """Loads config and cleans the companies dataset. Uses cleaned data to
+    generate new features.
```

Comment:
This actually needs to be changed for all of your cleaning files

Reply:
Do you mean a more involved docstring?

Reply:
"Loads config and cleans the **companies dataset**..."